prereise/gather/griddata/hifld/calculate/remap.py

```diff
@@ -0,0 +1,60 @@
+import copy
+
+
+def get_zone_mapping(zone):
+    """Generate dictionaries of zone using the zone.csv
+
+    :param pandas.DataFrame zone: the zone.csv data
+    :return: (*tuple*) -- a tuple of two dictionaries, where the first
+    dictionary is a dict mapping the STATE to its ID and the second is a dict
+    mapping a tuple of state and interconnect to its ID.
```

Comment:
the above 2 lines should be indented

Reply:
This function is getting zone mapping, but according to the doc strings here, none of the returns is zone related, which is ambiguous. I presume ID should be zone_id, STATE should be zone_name?

Reply:
Done. Updated the docstrings to reflect the headers found in the input csv.